apps/omg/mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule OMG.MixProject do
 
   defp deps() do
     [
-      {:ex_plasma, git: "https://github.com/omgnetwork/ex_plasma.git", ref: "867fee97dd930b47d22b5b32c4ef1751424c4206"},
+      {:ex_plasma, git: "https://github.com/omgnetwork/ex_plasma.git", ref: "8d51a76a106511b014caf51fdf70814684af9b9d"},
```

Comment:
what hash is this?

Reply:
it's hash of the master-v2. I'm not sure why it's changed. it can not find the old hash

Reply:
okay, we should revisit this but I think this works with `{:ex_plasma, "~> 0.2.0"},` too